README.md

```diff
@@ -0,0 +1,22 @@
+# Generic connects
+
+This project contains extensions for the base component "[Connect](https://github.com/th2-net/th2-conn)" component
+to support communication with the target system by using common protocols.
+
+Currently, it contains extensions for the following protocols:
++ [fix](conn-fix/README.md)
+
+## Build connects
+
+To build the "Connect" component you will need to have a Docker installed on your machine.
```

Comment:
To build the "Connect" component you will need to **have Docker** installed on your machine.